sum_types_generator/pubspec.yaml

```diff
@@ -1,18 +1,21 @@
 name: sum_types_generator
 description: sum_types and sum_types_generator packages together define a code generator enabling sum-types in Dart.
-version: 0.2.1+3
+version: 0.3.0
 homepage: https://github.com/werediver/sum-types-dart
 author: Raman Fedaseyeu <raman.fedaseyeu@gmail.com>
 
 environment:
-  sdk: '>=2.4.0 <3.0.0'
+  sdk: '>=2.12.0 <3.0.0'
 
 dependencies:
-  sum_types: ^0.2.0
-  meta: ^1.1.0
-  build: ^1.1.0
-  analyzer: '>=0.36.4 <0.42.0'
-  source_gen: ^0.9.2
+  analyzer: ^1.5.0
+  build: ^2.0.1
+  source_gen: ^1.0.0
+  sum_types: ^0.3.0
 
 dev_dependencies:
   dependency_validator:
+
+dependency_overrides:
+  sum_types:
+    path: ../sum_types
```

Comment:
You want to revert this. But I can't run `pub get` without because "0.3.0" isn't published yet.